### DISCUSSION TITLE
Logs

### DISCUSSION THREAD

**Discussion by manish7992** [artipie/artipie#956] [Q&A] [answered]
How to check the logs of Artipie ?

**Comment by g4s8** [comment, marked as answer]
Hi @Manish7992. To check Artipie logs you can start it and check standard output (see possible ways to start it: https://github.com/artipie/artipie/wiki). E.g. when you [start it in Docker container](https://github.com/artipie/artipie#quickstart) with `docker run --rm --name artipie -p 8080:8080 --user=artipie:artipie artipie/artipie:latest`, Docker CLI will print logs output right to stdout console. Same for [starting with JVM](https://github.com/artipie/artipie/wiki/Build-from-sources).
